src/main/java/uk/gov/hmcts/reform/sscs/model/Attendees.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.sscs.model;
```

Comment:
Can you move all the models into a subpackage, e.g.
`package uk.gov.hmcts.reform.sscs.model.hmc;`
As there are going to be different models for different endpoints

Reply:
will do